lte/gateway/c/core/oai/lib/s6a_proxy/proto_msg_to_itti_msg.cpp

```diff
@@ -139,15 +139,17 @@ void convert_proto_msg_to_itti_s6a_update_location_ans(
       ((msg.regional_subscription_zone_code_size() > MAX_REGIONAL_SUB) ?
            MAX_REGIONAL_SUB :
            (msg.regional_subscription_zone_code_size()));
-  uint8_t itr = 0;
+  uint8_t itr         = 0;
+  uint8_t reg_sub_idx = 0;
   while (itr < itti_msg->subscription_data.num_zcs) {
```

Comment:
use for loop and initialize and increment `itr` here.

Reply:
don't forget to remove `itr++` on line 155 once you make the change I suggested.